src/StaticLint.jl

```diff
@@ -144,11 +144,11 @@ function followinclude(x, state::State)
         path = get_path(x, state)
         if isempty(path)
         elseif hasfile(state.server, path)
-        elseif canloadfile(state.server, path)
+        elseif isabspath(path) && canloadfile(state.server, path)
```

Comment:
Wouldn't it make more sense to call `isabspath` right after the call to `get_path`, and then make sure we for example don't even call `hasfile` on a relative path? So essentially we only try the options where we `joinpath` with something if `get_path` returns a relative path.